queries.sql

```diff
@@ -51,8 +51,26 @@ SELECT owners.full_name, animals.name FROM owners LEFT JOIN animals ON owners.id
 
 SELECT species.name, COUNT(animals.id) FROM animals JOIN species ON animals.species_id = species.id GROUP BY species.name;
 
-SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id JOIN species ON animals.species_id = species.id WHERE owners.full_name = 'Jennifer Orwell'AND species.name = 'DigimON';
+SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id JOIN species ON animals.species_id = species.id WHERE owners.full_name = 'Jennifer Orwell'AND species.name = 'Digimon';
 
 SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
 
-SELECT owners.full_name FROM animals JOIN owners ON animals.owner_id = owners.id GROUP BY owners.full_name ORDER BY COUNT(animals.id) DESC LIMIT 1;
\ No newline at end of file
+SELECT owners.full_name FROM animals JOIN owners ON animals.owner_id = owners.id GROUP BY owners.full_name ORDER BY COUNT(animals.id) DESC LIMIT 1;
+
+SELECT animals.name FROM visits JOIN vets ON vet_id = vets.id JOIN animals ON animal_id = animals.id where vets.name = 'William Tatcher' ORDER BY date DESC LIMIT 1;
+
+SELECT COUNT(date) FROM visits JOIN vets ON vet_id = vets.id WHERE vets.name = 'Stephanie Mendez';
+
```

Comment:
- [ ] Kindly note that this query only returns counts of the total number of visits made by Stephanie Mendez. This query doesn't directly answer how many different animals she saw; it only gives you the total number of times she visited.
 - You can use this to directly answer the question
 
```
SELECT COUNT(DISTINCT animals.name) AS count_of_animals
FROM visits
LEFT JOIN animals ON animals.id = visits.animal_id
LEFT JOIN vets ON vets.id = visits.vet_id
WHERE vets.name = 'Stephanie Mendez';

```